### PR TITLE
JBIDE-27614: fix build to use tycho 2.2

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

**Comment by sbouchet** [comment]
testPR

File: common/plugins/org.jboss.tools.common.core/.classpath [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry exported="true" kind="lib" path="lib/jandex-2.0.4.Final.jar"/>
-	<classpathentry kind="output" path="target/classes"/>
```
Comment (jeffmaury) [inline common/plugins/org.jboss.tools.common.core/.classpath:7]: Not sure this is not a local change ?

**Review by jeffmaury — APPROVED** [review]
LGTM

File: common/plugins/org.jboss.tools.common.core/.classpath (hunk 1 above)
Comment (sbouchet) [reply common/plugins/org.jboss.tools.common.core/.classpath:7]: yes, looks like so, but does not influence the way the build is performed, as tycho does not use this  info but MANIFEST and build.properties.

**Pull request merged**